packages/playground/src/components/select_gateway_node.vue

```diff
@@ -4,34 +4,36 @@
     :rules="[validators.required('Gateway node is required.')]"
     #="{ props }"
   >
-    <v-autocomplete
-      label="Select gateway Node"
-      placeholder="Please select a domain."
-      :items="items"
-      item-title="domain"
-      return-object
-      v-bind="props"
-      @update:model-value="$emit('update:model-value', $event)"
-      :loading="items.length === 0 && loading"
-      :disabled="items.length === 0 && loading"
-      :model-value="$props.modelValue"
-    >
-      <template v-slot:append-item v-if="page !== -1">
-        <div class="px-4 mt-4">
-          <v-btn
-            block
-            color="secondary"
-            variant="tonal"
-            rounded="large"
-            size="large"
-            @click="loadNextPage"
-            :loading="loading"
-          >
-            Load More Gateway Nodes
-          </v-btn>
-        </div>
-      </template>
-    </v-autocomplete>
+    <input-tooltip tooltip="Refers to the process of choosing a specific node within a network to act as the gateway.">
```

Comment:
Creates a subdomain for your instance on the selected domain to be able to access your instance from the browser.